### ISSUE TITLE
pose configuration without bodypart's name and picture on MacOS

### ISSUE THREAD

**Issue by carlitomu** [sgoldenlab/simba#119]
Hi, 
I'have installed simba on my mac (version 10.13.6), following the instructions and using anaconda (https://github.com/sgoldenlab/simba/blob/master/docs/installation.md).
At the end of the installation process, in the prompt appear these message:

![Schermata 2021-06-09 alle 17 01 42](https://user-images.githubusercontent.com/85634193/121384190-540e8c80-c948-11eb-9a9f-d714c2c88489.png)

But, anyway, simba runs...

I open the program and try to create a project, but the pose configuration does not work correctly, it lacks of some fields.
I inserted the pose config name, the number of animals and bodyparts and I browse the image path and then clicking on "confirm" button nothing occured: the fields about the bodypart's name didn't appear in the window.

I have also tried to solve the problem with matplotlib (uninstalling and installing it again), but the pose configuration problem is always there.

Probably the problem is related to the MacOs system... 
Is there a simba version more suitable for MacOs System?
(I tried the installation process with and withou tensorflow: "simba-uw-tf==1.3.7" and "simba-uw-no-tf"

Thanks,
Carlo.

**Comment by sgoldenlab** [comment]
Hey @carlitomu - to run SimBA on a Mac, you will have to use the `simba-uw-tf-dev` version of SimBA. You can install it via `pip install `simba-uw-tf-dev` - can you give it a go and let me know if it boots up and you see all the buttons etc?

**Comment by carlitomu** [comment]
Thanks @sgoldenlab !
I created a new environment in anaconda and I used simba-uw-tf-dev version of SimBA, but the problems remains the sames.
The prompt continues to give this message:

<img width="1276" alt="Schermata 2021-06-09 alle 18 55 41" src="https://user-images.githubusercontent.com/85634193/121397778-f3d21780-c954-11eb-8936-7e1ea4ab3143.png">

Then in the pose configuration menu there'ren't the fields about the bodypart's name:

<img width="564" alt="Schermata 2021-06-09 alle 18 54 59" src="https://user-images.githubusercontent.com/85634193/121397965-25e37980-c955-11eb-9641-4cb3c0965d3d.png">

In the prompt appears this message

<img width="741" alt="Schermata 2021-06-09 alle 18 55 59" src="https://user-images.githubusercontent.com/85634193/121398009-3398ff00-c955-11eb-8535-f6461363585a.png">

An in SimBa this one:

<img width="737" alt="Schermata 2021-06-09 alle 18 55 16" src="https://user-images.githubusercontent.com/85634193/121398102-48759280-c955-11eb-8dfb-407ec6b2a806.png">

The number "4" I think it's related to the number of bodypart I inserted in the pose configuration window.
An it appears three time because I clicked three time on the "confirm" button.

So, this version of SimBa has the same problems with my mac.
